runtime/queries/ada/highlights.scm

```diff
@@ -135,41 +135,33 @@
   "use" @keyword.import
   "type" @keyword.import)
 
-(with_clause
-  "private" @keyword.import)
+(with_clause "private" @keyword.import)
 
-(with_clause
-  "limited" @keyword.import)
+(with_clause "limited" @keyword.import)
 
 (use_clause
   (_) @module)
 
 (with_clause
   (_) @module)
```

Comment:
Here you can see why special-casing anonymous nodes makes no sense.

Reply:
Rn, There's no reasonable query pattern to check for nodes without any child. I don't want to create a new predicate just for this use case

Reply:
Then I'm strongly in favor of leaving things as before.

Reply:
Although I don't quite see why it's impossible? Can't you just do `[ (anonymous_node) (named_node)]`?

Reply:
named nodes can have child.

You will need a lot more queries than just that.

Eg:
```
(foo (bar (baz (abc (xyz)))))
```

While anonymous nodes cannot have any more nested layers

Reply:
Also need to ignore other patterns that contains the named nodes :)
I'll try, but will only support 1-depth named nodes, no more than that

Reply:
>  will only support 1-depth named nodes

Oh yes, sorry in case that wasn't clear. I _only_ want
```
(with_clause (_) @module)
```
(or other named nodes), nothing else. But if it requires complications, then I'd argue for leaving named _and_ anonymous nodes alone.

Reply:
Added support for 

```
(foo (_) @capture)
```

This still stays
```
(foo
  (bar) @capture)
```

Reply:
Sorry, it's all or nothing for me. (Though I appreciate the effort.)

Reply:
> Sorry, it's all or nothing for me; that just makes it worse. Formatting should be based solely on the syntax, not the contents. (Though I appreciate the effort.)

Well now it's both then :)

Reply:
Only 1-depth. If a pattern is

```
(foo
  (bar
    (baz)))
```
it should stay as is

Reply:
I can live with this (although I still prefer the original, nicely nested, format). @ObserverOfTime is this an acceptable compromise?

Reply:
I'm not too fond of it…

Reply:
Me neither, but we have to compromise. Otherwise I'll invoke executive privilege.

Reply:
Let's just go back to the nested format.

Reply:
None of this is set in stone anyway. As long as the transformation is consistent and not relying on the input format (which is what I really care about), we can always change the format later.

Reply:
So which pattern should I be going for now :thinking:

Reply:
Your call; I can live with both and @ObserverOfTime can live with neither... (Or wait for @theHamsta to have yet a third opinion.)

Reply:
I'm gonna wait then. Both styles are fine to me

Reply:
Other considerations:

1. simpler implementation/faster formatting
2. smaller diff

Reply:
1. leads to bigger diffs

2. bit more implementations. But right now it's gone from 4->5 second for all files on GitHub actions, but still not too slow i guess

Reply:
Yeah, I don't think speed is relevant here (this was more a general comment). But I thought the diff against `HEAD` was _larger_ yesterday?

Reply:
Well yes, 2k lines larger. 

1k line was removed from this inlining. Another 1k line is from only `elixir/locals.scm`

Reply:
> Another 1k line is from only elixir/locals.scm

Which is irredeemable.

Reply:
Also why the file is blasted with `; format-ignore` cause that's the most I can care about it.

Reply:
Clearly the right move, and the reason we have that ("this is a dumpster fire; do not touch")

Reply:
> I can live with both and @ObserverOfTime can live with neither...

I can live with one node (of any kind) per line. At least it's consistent.

Reply:
I don't know of the mess I've created so I won't spend some time rebasing it, just gonna push :sob:

Reply:
Welp it's back now :)
If you want to bring back the inlining, just remove the `fixup: removal ...` and the commit following that